### DISCUSSION TITLE
Ability to upload badges to locked achievements

### DISCUSSION THREAD

**Discussion by Thoooreau** [RetroAchievements/RAWeb#1612] [Ideas]
So, instead of just being able to upload the unlocked version, that would result in this:

![charmander_lock](https://github.com/RetroAchievements/RAWeb/assets/40158645/d118ddbe-77b6-49a9-9101-5d9aae3ef45c) ⟶ ![charmander](https://github.com/RetroAchievements/RAWeb/assets/40158645/204841f3-12c9-4d22-b40a-baf22958f12c)

We would also be able to upload the locked version...

![charmander_shadow](https://github.com/RetroAchievements/RAWeb/assets/40158645/d5798b1b-fae9-4e3e-b2ff-d6979488317f)

So we could have this:

![charmander_shadow_lock](https://github.com/RetroAchievements/RAWeb/assets/40158645/84022070-5b17-4869-9a0d-70647cc66515) ⟶ ![charmander](https://github.com/RetroAchievements/RAWeb/assets/40158645/204841f3-12c9-4d22-b40a-baf22958f12c)

**Comment by Blazekickn** [comment]
I would prefer if the locked badge was in color. That would make it easier to show people clues, and also be easier on the site since we can just replace the current locked badge file.
